Sourcery/Parsing/Parser.swift

```diff
@@ -297,9 +297,22 @@ final class Parser {
             })
         }
 
-        //TODO: replace typealiases of typealiases
-        //typealias Foo = Int; typealias Bar = Foo
-        //TODO: replace typealiases in inherited types
+        //! if a typealias leads to another typealias, follow through and replace with final type
+        typealiasesByNames.forEach { _, alias in
+
+            var aliasNamesToReplace = [alias.name]
```

Comment:
shouldn't it be empty at the beginning? And I'm not sure I can get the following loop.